pom.xml

```diff
@@ -762,14 +762,19 @@
       <dependency>
         <groupId>org.json4s</groupId>
         <artifactId>json4s-jackson_${scala.binary.version}</artifactId>
-        <version>3.5.3</version>
+        <version>3.6.5</version>
         <exclusions>
           <exclusion>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>*</artifactId>
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.scala-lang.modules</groupId>
+        <artifactId>scala-xml_${scala.binary.version}</artifactId>
+        <version>1.2.0</version>
```

Comment:
Please move [scala-xml](https://github.com/apache/spark/blob/b73744a147d6e22167fbfecbe2892d94928cd5c2/LICENSE-binary#L415) to Apache License Version 2.0. `scala-xml` relicense under Apache 2.0 since [v1.1.1](https://github.com/scala/scala-xml/releases/tag/v1.1.1).

Reply:
Good catch. @igreenfield concretely that means the line for scala-xml in `LICENSE-binary` should move from the BSD 3-clause section to Apache License 2.0 section above. Also we need to copy the text from https://github.com/scala/scala-xml/blob/master/NOTICE to end of `NOTICE-binary`.